src/rpm/systemd-rpm-helper.in

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+
+command="${1:?}"
+shift
+
+command -v systemctl >/dev/null || exit 0
+
+case "$command" in
+    install-system-units)
+        systemctl --no-reload preset "$@"
+        ;;
+
+    install-user-units)
+        systemctl --no-reload preset --global "$@"
+        ;;
+
+    remove-system-units)
+        if [ -d /run/systemd/system ]; then
+            systemctl --no-reload disable --now "$@"
+        else
+            systemctl --no-reload disable "$@"
+        fi
+        ;;
+
+    remove-user-units)
+        systemctl --global disable "$@"
+
+        [ -d /run/systemd/system ] || exit 0
+
+        users=$(systemctl list-units 'user@*' --legend=no | sed -n -r 's/.*user@([0-9]+).service.*/\1/p')
```

Comment:
That seems to suggest that an option is missing in `list-units` to output the names of the units only.

Reply:
Yep. But it's a more complex thing: to get rid of the postprocessing, we'd need to have an option to print the instance values: I think that would make sense in general, but it'll require some design work. (We could have an option like name|prefix|instance, but maybe a different approach is more appropriate, like using percent-specifiers to specify a format to be resolved?). Once that's added, we can trivially update the call here.

Reply:
Using a generic option like git(1) does with its `--format=` option would be a possible solution. We could start with `--format=table` (the current output format) and `--format=name`. In the future, we could add support for percent-specifiers if the need arise.

BTW I'm not requesting this to be added to your PR.